test/e2e/arktos/kubectl/README.md

```diff
@@ -0,0 +1,81 @@
+When the test script sees a line of "Command: ###", it thinks the previous test case definition is complete.
+* "ExpectOutput: ###": this element is optional. When defined, the test will verify the command output contains the expected content. Multiple checks can be defined in one line, which will be verified one by one. Each check content can be strings concatenated by commas, which means the test expects these strings appear in the same line.
+For Instances, the following defintion means the test expects the output has a line containing both "AAA" and "BBB" and also has a line containing "CCC".
+```
+ExpectOutput: "AAA,BBB" "CCC"
+```
+* "ExpectFail: ###": this element is optional. If the value is set as "true", the test expects this command to have a non-zero exit code and report test failure if the command succeeds. By default or any other values, the test expects the command to succeed. Note that it means if you put the value as "yes", the test still expects the command to exit.
+* "TimeOut: ###": this element is optional. It defines the limit of number of seconds you expect the command to complete. The test fails if the command did not finish within the timeout value. The default timeout is configurable in config-kubectl-e2e-test.sh. If you don't want a test case undergo timeout check, set "TimeOut: 0"/
+* "RetryCount: ###": this element is optional. It defines the maximal number of retries before a test case is declared as failed. The default retry count is defined to 0, which means no retrial.
+* "RetryInterval: ###": this element is optional. It defines the number of seconds between two retries. The default retry count is set to 1 second.
+
+A line starting with "#" is regarded as comment and thus ignored. An empty line is also ignored. Any other formats of lines in the test case files will also be ignored, yet the test script will pop a waring.
+
+A test case definition can be as simple as just one line as below, which just checks the command's exit code is 0:
+```
+Command: ${kubectl} config view
+```
+
+Or, it can be as full-fledged as this one:
+```
+# the test is doomed to fail no matter how many times tried
+Command: ${kubectl} delete tenant non-existing-tenant
+ExpectFail: true
+ExpectOutput: "non-existing-tenant,deleted"
+Timeout: 10
+RetryCount: 3
+RetryInterval: 10
+```
+
+### Best Practice 
+
+Define a test case with as few lines as possible. It also make sense to avoid a long test case file and group related test cases into one file.
+
+Test is the best preventive medicine to code regression. I wish every bug fixing related to kubectl is accompanied by a test case addition.
+
+## What NOT to Expect
+
+As the test script is built from scratch and based on Linux Bash, the author expects the test is quick & lightweight, and adding new test cases is simple. But please **DO NOT** expect it to have:
+* Clearly Isolated Test Cases
```

Comment:
what does "clearly isolated" mean exactly?

Reply:
I added a paragraph as below to explain what I mean.